src/client/components/Button/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Button.styles.css';
+
+export const Button = (props) => {
+  return (
+    <button className="button-register-yourself" type="submit">
+      {props.text}
```

Comment:
Prefer `children` over custom props.

Reply:
not sure of using this, but I am trying

Reply:
Use `props.children`.
And for `propTypes` use `PropTypes.node`.